src/coreclr/jit/ssabuilder.cpp

```diff
@@ -1284,19 +1281,11 @@ void SsaBuilder::RenameVariables()
 //
 void SsaBuilder::Build()
 {
-#ifdef DEBUG
-    if (m_pCompiler->verbose)
-    {
-        printf("*************** In SsaBuilder::Build()\n");
-    }
-#endif
+    JITDUMP("*************** In SsaBuilder::Build()\n");
```

Comment:
In general, this is fine. Note, however, that the older form has one useful feature: you can set an unconditional line-based breakpoint on the "printf" that only activates when debugging a function where you've enabled JitDump (so `verbose` is true).

Reply:
Ah, that makes sense... I'll avoid making this change in the future.